test.rb

```diff
@@ -0,0 +1,17 @@
+a = 1
+  class AAA
+    def initialize
+      @a
+    end
+  
+    def a
+      @a
+    end
+  
+      def foo
+        bar
+      end
+  end
+  
+  a = ['x','y','z']
```

Comment:
__[staging][rubocop]__
* Inconsistent indentation detected.
* Use `%w` or `%W` for an array of words. [:ref](https://github.com/bbatsov/ruby-style-guide#percent-w)
* Space missing after comma. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-operators)

<!-- Comment by [SideCI](https://sideci.com) -->